01_fib/alanhillgemann/main_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("0\n1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("expected %v, actual %v", expected, actual)
+	}
+}
+
+func TestFib(t *testing.T) {
+	// You can create many test cases succinctly with an anonymous struct array
```

Comment:
I'm not sure this comment is necessary. It seems more like a pro coding tip, rather than providing extra information for this test.

If you wanted to keep it, perhaps you could reword to something like: `// Create multiple test cases inside an anonymous struct array`

Reply:
Thanks for your review, I've removed the comments